.github/workflows/build.yml

```diff
@@ -0,0 +1,265 @@
+            APP_ENV: test
+            DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?serverVersion=${{ matrix.mysql }}"
+
+        steps:
+            -
+                uses: actions/checkout@v3
+
+            -
+                name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: "${{ matrix.php }}"
+                    extensions: intl
+                    tools: flex,symfony
+                    coverage: none
+
+            -
+                name: Setup Node
+                uses: actions/setup-node@v3
+                with:
+                    node-version: "${{ matrix.node }}"
+
+            -
+                name: Shutdown default MySQL
+                run: sudo service mysql stop
+
+            -
+                name: Setup MySQL
+                uses: mirromutth/mysql-action@v1.1
+                with:
+                    mysql version: "${{ matrix.mysql }}"
+                    mysql root password: "root"
+
+            -
+                name: Output PHP version for Symfony CLI
+                run: php -v | head -n 1 | awk '{ print $2 }' > .php-version
+
+            -
+                name: Install certificates
+                run: symfony server:ca:install
+
+            -
+                name: Run webserver
+                run: (cd tests/Application && symfony server:start --port=8080 --dir=public --daemon)
+
+            -
+                name: Run Chrome Headless
+                run: google-chrome-stable --enable-automation --disable-background-networking --no-default-browser-check --no-first-run --disable-popup-blocking --disable-default-apps --allow-insecure-localhost --disable-features=Translate --disable-extensions --no-sandbox --enable-features=Metal --headless --remote-debugging-port=9222 --window-size=2880,1800 https://127.0.0.1 > /dev/null 2>&1 &
+
+            -
+                name: Get Composer cache directory
+                run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+                id: composer-cache
+
+            -
+                name: Cache Composer
+                uses: actions/cache@v3
+                with:
+                    path: ${{ steps.composer-cache.outputs.dir }}
+                    key: ${{ runner.os }}-php-${{ matrix.php }}-composer-${{ hashFiles('**/composer.json **/composer.lock') }}
+                    restore-keys: |
+                        ${{ runner.os }}-php-${{ matrix.php }}-composer-
+
+            -
+                name: Configure global composer
+                if: matrix.symfony != ''
+                run: composer global config --no-plugins allow-plugins.symfony/flex true
+
+            -
+                name: Restrict Sylius version
+                if: matrix.sylius != ''
+                run: composer require "sylius/sylius:${{ matrix.sylius }}" --no-update --no-scripts --no-interaction
+
+            -
+                name: Fix build on Sylius 1.9
```

Comment:
Unnecessary step, since we won't support sylius < 1.10.
We could even bump the requirement to ^1.11,  since it's a new package and no old projects are affected by it, and nobody should start new projects with old versions of Sylius anyway,